hackathon_site/dashboard/frontend/src/assets/abstracts/_exports.scss

```diff
@@ -3,5 +3,5 @@
 :export {
     //  Exports to plug into MUI's theming system
     primary: color(primary);
-    secondary: color(secondary);
+    secondary: color(red);
```

Comment:
Are you sure you want to do this? This will make MUI use red for all secondary colors (backgrounds, buttons, font colors, etc).

I would instead set the warning and error palettes to be what you want: https://material-ui.com/customization/palette/. You can define them here, then add them to the theme in App.js